lecture/views.py

```
@@ -86,20 +86,68 @@ def lecture_pdf(request: HttpRequest,slug:str,pdf_id: int) -> HttpResponse:
+@require_http_methods(["GET", "POST"])
+def edit_lecture(request: HttpRequest,course_slug:str,lecture_slug:str) -> HttpResponse:
+    lecture = get_object_or_404(Lecture, slug=lecture_slug) if lecture_slug else None
+    if request.method == "POST":
+       
+        
+        form = LectureForm(request.POST, request.FILES, instance=lecture)
+        if form.is_valid():
+            form.save()
+            messages.success(request, "Lecture saved successfully!")
+            return redirect("lecture_home",slug=course_slug)
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)